nav2_mppi_controller/src/critics/cost_critic.cpp

```diff
@@ -153,27 +171,36 @@ void CostCritic::score(CriticData & data)
       // The footprintCostAtPose will always return "INSCRIBED" if footprint is over it
       // So the center point has more information than the footprint
       if (!worldToMapFloat(Tx, Ty, x_i, y_i)) {
-        if (!is_tracking_unknown_) {
-          traj_cost = collision_cost_;
-          trajectory_collide = true;
-          break;
-        }
-        pose_cost = 255.0f;  // NO_INFORMATION in float
+        pose_cost = nav2_costmap_2d::NO_INFORMATION;
```

Comment:
Revert; if we're not tracking unknown space, this is a collision case and we can skip scoring the rest of this trajectory